modules/akkaHttp/src/main/scala/ru/tinkoff/tschema/akkaHttp/Serve.scala

```diff
@@ -141,6 +141,10 @@ private[akkaHttp] trait ServeInstances extends ServeFunctions with ServeInstance
     parameterMap.map(_.contains(name[name]))
   )
 
+  implicit def queryMap[name <: Symbol: Witness.Aux, x, In <: HList] = serveAdd[AllQuery[name], In, name, Map[String, String]] {
```

Comment:
I'm not sure if Witness.Aux is required here since we need the name only to assign result to parameter and not to extract some data from RequestContext

Reply:
So, we can probably leave the type without specifying boundaries. `[queryMap[name, x, In <: HList]]`